internal/config/config_test.go

```diff
@@ -45,7 +45,9 @@ var (
 			Type:    "",
 		},
 		StorageConfig: service.StorageConfig{
-			RulesPath: "rules_path",
+			RulesPath:               "rules_path",
+			RemoteConfigurationPath: "remote_configurations",
+			ClusterMappingPath:      "./tests/rapid-recommendations/cluster-mapping.json",
```

Comment:
is this path relative to the CWD or to the this file?

Reply:
It is relative to CWD. Is that an issue?

Reply:
depends, I do not know if you can run the go tests from any folder or you need to be in a special one. In the first case, this test may fail if run from a different folder

Reply:
IMO we don't really take into account such option in general - when I try to run tests outside of service directory, it complains about missing `go.mod`. However, with `make tests` as a standard way we run tests, it's basically not possible to run tests outside that directory (`-C` argument is an option here, but it changes working directory, so it works correctly).